content/en/docs/faq/kubed.md

```diff
@@ -73,3 +73,5 @@ spec:
     kind: Issuer
     group: cert-manager.io
 ```
+
+[CertificateSecretTemplate]: ../../reference/api-docs/#cert-manager.io/v1alpha3.CertificateSecretTemplate
```

Comment:
We should link to the `v1` API version as the others are deprecated.

Reply:
Thanks for the review. I've updated the link.